Example/index.android.js

```diff
@@ -1,53 +1,88 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
-
-import React, { Component } from 'react';
+import React, { Component } from 'react'
```

Comment:
I feel like we should just unify this with the iOS example (i.e., rename the other one to index.js and remove this file).